README.md

```diff
@@ -56,6 +56,29 @@ composer require --dev friendsofphp/php-cs-fixer
 For more details and other installation methods, see
 [installation instructions](./doc/installation.rst).
 
+### Run with Docker
+
+You can take a ready built docker image to run ``php-cs-fixer``.
+
+```console
+docker run -v $(pwd):/code ghcr.io/php-cs-fixer/php-cs-fixer:$FIXER_VERSION fix src
```

Comment:
we could also say `php-cs-fixer:^3-php8.3 fix src` and avoid teaching user about extra env var

or have it like `${FIXER_VERSION:-^3-php8.3}`.

Reply:
Can we improve it later to not block this PR longer?

Reply:
it's one-liner to apply, even from diff view...
if we insist, we can externalise this to separated PR

Reply:
It's only because I already wrote "one step before merge" (twice) and from @hgiesenow's perspective it may be frustrating to do changes back and forth, depending on who comments 😅. I can address your concerns later, these are not crtitical.

Reply:
I added a default as suggested. That will improve acceptance and keeps it flexibel.